felix/logutils/logutils.go

```diff
@@ -140,10 +140,6 @@ func ConfigureLogging(configParams *config.Config) {
 	// hook above to fan out logs to multiple destinations.
 	log.SetOutput(&logutils.NullWriter{})
 
-	// Since we push our logs onto a second thread via a channel, we can disable the
```

Comment:
This was causing a race in Felix that's hard to fix because the lock is inside logrus.  I don't think the lock is very heavily contended so it's probably no big deal to re-enable it